doc/nrf/app_dev/board_names.rst

```diff
@@ -125,6 +125,8 @@ Also see the :ref:`zephyr:boards` section in the Zephyr documentation.
 |                   |            | :ref:`nrf52840dk <zephyr:nrf52840dk_nrf52811>`      | ``nrf52840dk/nrf52811``                                               |
 +-------------------+------------+-----------------------------------------------------+-----------------------------------------------------------------------+
 | nRF52840 Dongle   | PCA10059   | :zephyr:board:`nrf52840dongle <nrf52840dongle>`     | ``nrf52840dongle/nrf52840``                                           |
+|                   |            |                                                     |                                                                       |
+|                   |            |                                                     | ``nrf52840dongle/nrf52840/bare`` (without nRF5 SDK bootloader)        |
```

Comment:
You need to edit lines 48-61 in this file given this new variant. Please check chat for more details of what needs changing.